package.json

```diff
@@ -1,794 +1,829 @@
-					"highContrast": "#ff8c00"
-				}
-			},
-			{
-				"id": "coq.termReference",
-				"description": "Color of name references in terms.",
-				"defaults": {
-					"dark": "#7cfc00",
-					"light": "#32cd32",
-					"highContrast": "#32cd32"
-				}
-			},
-			{
-				"id": "coq.termType",
-				"description": "Color of types in terms.",
-				"defaults": {
-					"dark": "#f08080",
-					"light": "#ff7f50",
-					"highContrast": "#ff7f50"
-				}
-			},
-			{
-				"id": "coq.termVariable",
-				"description": "Color of variables in terms.",
-				"defaults": {
-					"dark": "#7fff00",
-					"light": "#20b2aa",
-					"highContrast": "#20b2aa"
-				}
-			},
-			{
-				"id": "coq.debugMessage",
-				"description": "Color of debug messages.",
-				"defaults": {
-					"dark": "#9370d8",
-					"light": "#daa520",
-					"highContrast": "#daa520"
-				}
-			},
-			{
-				"id": "coq.errorMessage",
-				"description": "Color of error messages.",
-				"defaults": {
-					"dark": "#ff0000",
-					"light": "#ff0000",
-					"highContrast": "#ff0000"
-				}
-			},
-			{
-				"id": "coq.warningMessage",
-				"description": "Color of warning messages.",
-				"defaults": {
-					"dark": "#ffff00",
-					"light": "#ffff00",
-					"highContrast": "#ffff00"
-				}
-			},
-			{
-				"id": "coq.moduleKeyword",
-				"description": "Color of module keywords.",
-				"defaults": {
-					"dark": "#6495ed",
-					"light": "#6495ed",
-					"highContrast": "#6495ed"
-				}
-			},
-			{
-				"id": "coq.tacticKeyword",
-				"description": "Color of tactic keywords.",
-				"defaults": {
-					"dark": "#6495ed",
-					"light": "#6495ed",
-					"highContrast": "#6495ed"
-				}
-			},
-			{
-				"id": "coq.tacticPrimitive",
-				"description": "Color of tactic primitives.",
-				"defaults": {
-					"dark": "#5f9ea0",
-					"light": "#5f9ea0",
-					"highContrast": "#5f9ea0"
-				}
-			},
-			{
-				"id": "coq.tacticString",
-				"description": "Color of tactic strings.",
-				"defaults": {
-					"dark": "#ffa500",
-					"light": "#ffa500",
-					"highContrast": "#ffa500"
-				}
-			}
-		]
-	},
-	"main": "./client/out/src/extension",
-	"scripts": {
-		"vscode:prepublish": "npm run compile",
-		"compile": "cd client && npm run compile && cd ../server && npm run compile && cd ../html_views && npm run compile",
-		"postinstall": "cd client && npm install && cd ../server && npm install && cd ../html_views && npm install && cd .."
-	},
-	"devDependencies": {
-		"typescript": "^3.7.4",
-		"ts-prune": "^0.3.0",
-		"vsce": "^1.71.0"
-	},
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/coq-community/vscoq.git"
-	}
+  "name": "vscoq",
+  "displayName": "VSCoq",
+  "description": "An IDE for the Coq Proof Assistant",
+  "author": {
+    "name": "Christian J. Bell",
+    "email": "siegebell@gmail.com",
+    "url": "http://people.csail.mit.edu/cj/"
+  },
+  "version": "0.3.0",
+  "publisher": "maximedenes",
+  "license": "MIT",
+  "engines": {
+    "vscode": "^1.38.0"
```

Comment:
I changed this back to `1.23.0`, ran `make clean && make`, and it succeeded. What did I do wrong?

Reply:
Try:

```
$ git clean -dfx
$ npm i
$ npm run compile
```

IIRC, it downloads an outdated definition file and fails to build.

Reply:
Oh, right, so it means there's a problem with the `clean` target. Not sure if introduced by this PR.

Reply:
I hadn't noticed the Makefile until now. Seems brittle.

Reply:
> Yeah, the Makefile is junk; we should throw it out.

Why? What's wrong with it?

Reply:
The primary purpose of a Makefile is to incrementally build artifacts that are out-of-date. This Makefile has exactly one real dependency:
```
compile: node_modules
```

Obviously, this is incorrect, and `compile` should rebuild if anything in `client` or `server` changes. Besides, we now have incremental builds via `npm`: `tsc -b`. Makefiles are really ill-suited to js/ts projects, where npm is the build-tool of choice.

Reply:
Fair enough, but let's address this in another PR. In particular, we need to add a call to `vsce` from `npm` before removing the Makefile.

Meanwhile, we can just adjust the directories to be erased. I'll push to this branch.

Reply:
No, `vsce package` is only useful to the extension publisher (namely, you): you shouldn't make contributors suffer slower build times for no benefit.

Reply:
Yes, but there should be a build target for it, it does not have to be the default one.

Reply:
Why though? Just use `vsce publish patch` or similar directly -- why do you want a `vsce package`, and an even longer `npm run package`?